### ISSUE TITLE
MessageBar: [a11y] MessageType.error should use aria-live="assertive" instead of "polite"

### ISSUE THREAD

**Issue by vashitaArora** [OfficeDev/office-ui-fabric-react#843]
Error Message should be announced preemptively. we should either have an option to ourselves specify the aria-live value or errors shoudl by default be assertive

**Comment by mdahamiwal** [comment]
Should be a simple fix check http://codepen.io/mdahamiwal/pen/XpROQP
